test/testLiveAsTemplate.js

```diff
@@ -124,6 +124,10 @@ describe('CLI interface, oghliner as a template', function() {
     .then(function() {
       assert.doesNotThrow(fse.statSync.bind(fse, 'index.html'));
       assert.doesNotThrow(fse.statSync.bind(fse, 'offline-worker.js'));
+    })
+    .catch(function(err) {
+      console.log('Error: ' + err);
+      throw err;
```

Comment:
Yes, for some reason mocha doesn't print to the console the rejection error in this case... So I've explicitly added logging, otherwise we don't know why the test fails when it fails.

I've tried writing a simpler test:
```
describe('a', function() {
  it('test', function() {
    return Promise.reject(new Error('message'));
  });
});
```

And it logs:
```
  a
    1) test


  0 passing (12ms)
  1 failing

  1) a test:
     Error: message
      at Context.<anonymous> (test/testRejection.js:3:27)
```

So I'll look into it more.